### ISSUE TITLE
Return null

### ISSUE THREAD

**Issue by wassy92x** [Typescript-TDD/ts-auto-mock#207]
Hello,

I have the problem, that in one of my tests I need the return value "null".
My Interface looks like these:
```
interface IFooBar {
  readonly id: string;
}
interface IFoo {
  readonly bar: IFooBar;
}
```

When I create a new mock with
```
createMock<IFoo>({
  bar: null
});
```

The resulting object is always:
```
{
  bar: {
    id: ""
  }
}
```

Is it possible somehow to return null instead?

**Comment by Pmyl** [comment]
Hi!
This is an interesting bug, I think it’s a problem related on how we merge the properties, it could be that “null” is interpreted as “ignore it”.

I’ll check when I get home and let you know asap

**Comment by Pmyl** [comment]
@wassy92x I've found the bug in the code, I'll fix it later today and create a new version

**Comment by wassy92x** [comment]
nice! Thanks for your fast response and your support!

**Comment by Pmyl** [comment]
@wassy92x I've fixed it and released version 1.5.0 that is also containing other bug fixes and few new supported types!

Try to update and let us know
